docs/features/techdocs/getting-started.md

```diff
@@ -62,8 +61,7 @@ Navigate to `packages/backend` of your Backstage app, and install the
 
 ```bash
 cd my-backstage-app/
-cd packages/backend
-yarn add @backstage/plugin-techdocs-backend
+yarn --cwd packages/backend add @backstage/plugin-techdocs-backend
```

Comment:
I kind of like breaking this into two smaller, simpler lines and being more explicit. :)
But I appreciate you suggesting `--cwd`, didn't know about it. Could be useful in scripts and such.

Reply:
Yes, my thinking here is that I tend to never leave root directory of project if I can. :)